app/core/WalletConnect/WalletConnect2Session.test.ts

```diff
@@ -169,19 +169,19 @@ describe('WalletConnect2Session', () => {
     jest.spyOn(Linking, 'openURL').mockReturnValue(Promise.resolve());
   });
 
-  it('should initialize correctly in constructor', () => {
+  it('initializes correctly in constructor', () => {
     expect(session).toBeTruthy();
     expect((session as any).topicByRequestId).toEqual({
       '1': mockSession.topic,
     });
   });
 
-  it('should set deeplink correctly', () => {
+  it('sets deeplink correctly', () => {
     session.setDeeplink(false);
     expect((session as any).deeplink).toBe(false);
   });
 
-  it('should handle request correctly and reject invalid chainId', async () => {
+  it('handles request correctly and rejects invalid chainId', async () => {
```

Comment:
S: focus on the result and avoid weasel words like "handle" that caries no meaning and all meanings at the same time. Favour precise language. Here the request working is part of the implementation. The fact that it rejects invalid chainid is the expectation, use this:
```suggestion
  it('rejects invalid chainId', async () => {
```